src/sentry/web/api.py

```diff
@@ -306,6 +307,9 @@ def post(self, request, **kwargs):
             # bubble up as an APIError.
             data = None
 
+        if getattr(settings, 'ENABLE_PUBSUB', False):
+            pubsub.put('requests', data)
```

Comment:
Can't think off the top of my head... do we need to copy the bytes here before we pass it along to another thread? I _think_ this is immutable.

/cc @tkaemming @mitsuhiko

Reply:
I guess it's immutable? But if not we need to serialize it out here.

Reply:
For more context, `data` here is a byte array directly from `request.body`. We're intending to forward it along to a redis pubsub channel before it's decoded normally. I _think_ this will be safe, but just want to make sure with someone who knows more about python than me.

Reply:
Or we just do `data[:]` to be safe and have a pure copy of it.

Reply:
Oh sorry, I thought this was decode JSON data already. That's fine then.

Reply:
> That's fine then.

As is? Or to copy the bytes into the thread?

Reply:
No need to copy, it's immutable.